PaymentSucceful/PaymentSucceful.css

```diff
@@ -1,59 +1,36 @@
-
 * {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
   font-family: "Poppins", sans-serif;
 }
-body {
-  min-height: 100vh;
+
```

Comment:
We need the background
`  background: linear-gradient(
    90deg,
    rgba(160, 106, 186, 0.1) 0%,
    rgba(255, 255, 255, 0) 33.33%
      );`